org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/completion/LSContentAssistProcessor.java

```diff
@@ -137,6 +137,9 @@ public ICompletionProposal[] computeCompletionProposals(ITextViewer viewer, int
 								if (isIncomplete) {
 									anyIncomplete.set(true);
 								}
+							}).exceptionally(t -> {
+								LanguageServerPlugin.logError("'%s' LS failed to compute completion items.".formatted(w.serverDefinition.label), t); //$NON-NLS-1$
```

Comment:
Do we not need an if such as
`			if (!CancellationUtil.isRequestCancelledException(e)) { // do not report error if the server has cancelled the request
`here as well?

Reply:
Fair point :-)